build/css.mk

```diff
@@ -12,12 +12,14 @@ dist/css/parts/%: src/css/parts/%
 		build/install.sh 644 $< $@
 
 # CSS rules
-dist/css/min/black-highlighter.min.css: src/css/black-highlighter.css $(BUILD_SOURCES) $(CSS_SOURCES) node_modules
+dist/css/min/black-highlighter.min.css: src/css/black-highlighter.css $(CSS_SOURCES) node_modules
 		build/install.sh 644 $< dist/css/black-highlighter-imports.css
 		bun postcss dist/css/black-highlighter-imports.css --config build --env development -o dist/css/black-highlighter.css
 		bun postcss dist/css/black-highlighter-imports.css --config build --env production -o $@
+		touch $@
 
-dist/css/min/normalize.min.css: src/css/normalize.css $(BUILD_SOURCES) node_modules
+dist/css/min/normalize.min.css: src/css/normalize.css node_modules
```

Comment:
Why were these removed? If a build process is updated then those build rules need to be rerun, because it might be different now, right?

Reply:
Meant to leave a comment. I wasn't sure why these were included but that makes sense! I've added them back in.